repose-aggregator/core/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -149,6 +154,20 @@ class ReposeJettyServer(val clusterId: String,
     //Have to coerce the stuff here, because it makes it happier
     s.setConnectors(connectors.asInstanceOf[Array[Connector]])
 
+    val security: ConstraintSecurityHandler = new ConstraintSecurityHandler()
+    s.setHandler(security)
+
+    //Adds a constraint that will throw 405 when method does not exist
+    val constraint = new Constraint()
+    constraint.setName("methods")
+
+    val mapping = new ConstraintMapping()
+    mapping.setPathSpec("/*")
+    mapping.setMethodOmissions(HttpComponentFactory.values.map(method => method.toString))
```

Comment:
method => method.toString could be replaced by _.toString

In this simple case, I'd suggest it. It kind of comes down to preference, though.